tensorboard/webapp/webapp_data_source/tb_feature_flag_data_source_types.ts

```diff
@@ -34,3 +34,5 @@ export abstract class TBFeatureFlagDataSource {
 export const EXPERIMENTAL_PLUGIN_QUERY_PARAM_KEY = 'experimentalPlugin';
 
 export const GPU_LINE_CHART_QUERY_PARAM_KEY = 'fastChart';
+
+export const SCALARS_BATCH_SIZE_PARAM_KEY = 'scalarsBatchSize';
```

Comment:
If you use a bit more generic name such as "maxDataBatchSize`, then you can reuse this under timeseries plugin.

Reply:
Should it only tune scalars-related calls in Time Series dashboard?  In that case I think the name is fine (don't have a strong opinion though - 'maxDataBatchSize' might be just as good).

I don't think we should use this to tune anything non-scalars related since we don't (yet) know whether other things have similar performance characteristics.  It's entirely possible we will want a different value for batching histograms or blobs, for example.

Reply:
Fair. SG.